AllyAbilities.php

```diff
@@ -221,6 +221,13 @@ function DestroyAlly($player, $index, $skipDestroy = false, $fromCombat = false,
     CollectBounties($player, $index);
     IncrementClassState($player, $CS_NumAlliesDestroyed);
   }
+
+  if ($fromExploit) {
```

Comment:
why did you have special case for exploit ?

Reply:
i had a different solution before coming to this one. i must have missed this spot in refactoring. that has now been removed